src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -203,17 +234,10 @@ static boolean hasOcr(CaseDetails theCase) {
             .orElse(invalid("Missing " + dateField));
     }
 
-    /**
-     * Used in createCase callback only. Can be reviewed later and with BPS-746 - revised the usage.
-     * @param theCase from CCD.
-     * @return Validation of it
-     */
     @SuppressWarnings("unchecked")
-    public static Validation<Object, List<Map<String, Object>>> getOcrData(CaseDetails theCase) {
+    public static Optional<List<Map<String, Object>>> getOcrData(CaseDetails theCase) {
```

Comment:
Doesn't the change of this method affect the validation? As far as I understand from the code, if OCR data is missing then `getOcrDataFields` will return `valid(null)`, instead of `invalid("Missing OCR data")`.